frontend/src/pages/LocationsPage.js

```diff
@@ -0,0 +1,46 @@
+import useLocations from "../hooks/useLocations";
+import LocationList from "../components/LocationList";
+import {IconButton} from "@material-ui/core";
+import MapIcon from '@material-ui/icons/Map';
+import ListIcon from '@material-ui/icons/List';
+import {useState} from "react";
+import LocationMap from "../components/LocationMap";
+import styled from "styled-components/macro";
+
+export default function LocationsPage() {
+
+    const locations = useLocations()
+    const [mapIsEnabled, setMapIsEnabled] = useState(false)
+
+    const switchView = (event, value) => {
+        event.preventDefault()
+        setMapIsEnabled(value)
+    }
+
+    return (
+        <div>
```

Comment:
maybe move both components to separate components? Makes it easier to understand what is happening here